razorpay-payments.php

```diff
@@ -76,6 +77,20 @@ public function __construct()
             $this->msg['message'] = '';
             $this->msg['class'] = '';
 
+            // Supports woocommerce subscriptions
+            $this->supports = array(
+                'products',
+                'subscriptions',
+                'subscription_cancellation',
+                'subscription_suspension',
+                'subscription_reactivation',
+                'subscription_amount_changes',
```

Comment:
1. This was already defined on line 71
2. We don't support all of these. Just stick to `subscriptions` for now. I'm not seeing any code for anything else (cancellation/suspension etc)